examples/wheel/wheel_test.py

```diff
@@ -18,18 +18,23 @@
 import unittest
 import zipfile
 
+from python.runfiles import runfiles
+
 
 class WheelTest(unittest.TestCase):
     maxDiff = None
 
-    def test_py_library_wheel(self):
-        filename = os.path.join(
-            os.environ["TEST_SRCDIR"],
-            "rules_python",
-            "examples",
-            "wheel",
-            "example_minimal_library-0.0.1-py3-none-any.whl",
+    def setUp(self):
+        super().setUp()
+        self.runfiles = runfiles.Create()
+
+    def _get_path(self, filename):
+        return self.runfiles.Rlocation(
+            os.path.join("rules_python/examples/wheel", filename)
         )
```

Comment:
nit: we could improve this by passing `pathlib.Path` around.

```suggestion
        path = pathlib.Path(
            self.runfiles.Rlocation(
                os.path.join("rules_python/examples/wheel", filename)
            )
        )
        self.assertTrue(path.exists(), f"The path for {filename} does not exist")
        return path
        
```

Reply:
I tried this, but then realized the path is just immediately passed into ZipFile, so any of the additional functionality of pathlib isn't really used. I did update this to check for existence, though, which should help hint any any file not found errors in the future.